packages/synapse-react-client/src/synapse-queries/dataaccess/useDataAccessSubmission.ts

```diff
@@ -199,16 +199,19 @@ export function useSubmitDataAccessRequest(
       return SynapseClient.submitDataAccessRequest(request, accessToken!)
     },
     onSuccess: async (data, variables, ctx) => {
-      // Invalidate the status of the relevant AR
-      await queryClient.invalidateQueries({
-        queryKey: keyFactory.getAccessRequirementStatusQueryKey(
-          variables.accessRequirementId,
-        ),
-      })
-      // Invalidate all searches, in case it was an AR reviewer who created this submission
-      await queryClient.invalidateQueries({
-        queryKey: keyFactory.searchDataAccessSubmissionQueryKey(),
```

Comment:
no longer need to invalidate searchDataAccessSubmissionQueryKey ?

Reply:
The new `getDataAccessSubmissionQueryKey` includes `searchDataAccessSubmissionQueryKey` in its scope when used as an invalidation key